### PR TITLE
Correctly shuffle training data+labels so that training works

### PR DESCRIPTION
**Pull request by gbuesing**
You weren't shuffling your data with your labels -- do that and your training setup works.

### CODE REVIEW

**Comment by bararchy** [comment]
I see, Merged.
